python/pyspark/ml/clustering.py

```diff
@@ -330,6 +357,20 @@ class KMeans(JavaEstimator, HasFeaturesCol, HasPredictionCol, HasMaxIter, HasTol
     >>> df = spark.createDataFrame(data, ["features"])
     >>> kmeans = KMeans(k=2, seed=1)
     >>> model = kmeans.fit(df)
+    >>> summary = model.summary
```

Comment:
Can we make this doc test the same as the other clustering classes? We should test `model.hasSummary` before and after save/load as was done in the other implementations. Also, we need to add a test to `tests.py` for this summary (see that file and reference the other summary tests).